integration-tests/src/main/java/io/druid/testing/ConfigFileConfigProvider.java

```diff
@@ -61,10 +64,18 @@ private void loadProperties(String configFile)
     }
     routerHost = props.get("router_host") + ":" + props.get("router_port");
     brokerHost = props.get("broker_host") + ":" + props.get("broker_port");
+    historicalHost = props.get("historical_host") + ":" + props.get("historical_port");
     coordinatorHost = props.get("coordinator_host") + ":" + props.get("coordinator_port");
     indexerHost = props.get("indexer_host") + ":" + props.get("indexer_port");
-    middleManagerHost = props.get("middle_manager_host");
+    middleManagerHost = props.get("middlemanager_host");
```

Comment:
if it has always been wrong then, it is fine. This change gave me the impression that we accidendally changed a property name.